src/hotspot/share/runtime/os.cpp

```diff
@@ -1176,12 +1176,12 @@ void os::print_location(outputStream* st, intptr_t x, bool verbose) {
 
 // Looks like all platforms can use the same function to check if C
 // stack is walkable beyond current frame.
```

Comment:
This comment is somewhat weird and it - and the one at the prototype in os.hpp - could do with some massaging. Buts its fine to do this in a different RFE.

Reply:
yes...